plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Table, TableColumn } from '@backstage/core';
+import { Box, Typography } from '@material-ui/core';
+import RetryIcon from '@material-ui/icons/Replay';
+import React from 'react';
+import { useConsumerGroupFromEntity } from './useConsumerGroupsFromEntity';
+import { useConsumerGroupOffsets } from './useConsumerGroupOffsets';
+
+export type TopicPartitionInfo = {
+  topic: string;
+  partitionId: number;
+  topicOffset: string;
+  groupOffset: string;
+};
+
+const generatedColumns: TableColumn[] = [
+  {
+    title: 'Topic',
+    field: 'topic',
+    highlight: true,
+    render: (row: Partial<TopicPartitionInfo>) => {
+      return <>{row.topic ?? ''}</>;
+    },
+  },
+  {
+    title: 'Partition',
```

Comment:
Is this the # of partitions for the topic? If so probably be plural.

Reply:
No, that's the partition index within the topic (see screenshot above)